index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <header id="hello-microverse">Hello Microverse</header>
```

Comment:
- [ ] Great job so far, but there is something you still need to work on so that your project looks fantastic; you should only use 'h1' instead of a 'header' tag, and I observed that you forgot to include '!' at the end of Hello Microverse ; please make the necessary changes to your project. :wink:  :wink:

Reply:
thanks very much, for your honest review I'm going to correct my mistakes.